### PR TITLE
Feature #145 pagination

### LINKED ISSUE #145: Agregar paginación a classroom
Hoy en día la tab de estudiantes muestra a todos, debería estar paginada.

### PR DESCRIPTION
**Pull request by fedescarpa**
merge after https://github.com/mumuki/mumuki-classroom-api/pull/116

fixes #145

### CODE REVIEW

File: src/scripts/controllers/guide-progress-controller.js [hunk 1]
```diff
@@ -41,12 +47,7 @@ angular
       return _.get(notifications, guideProgress.student.uid, []);
     }
 
-    $scope.availableSortingCriterias = [
-      { type: 'messages', properties: [$scope.notifications, 'student.last_name', 'student.first_name']},
-      { type: 'name', properties: ['student.last_name', 'student.first_name']},
-      { type: 'progress', properties: ['stats.total', 'passedAverage()', 'student.last_name', 'student.first_name']},
-      { type: 'last_submission_date', properties: ['-lastSubmission().created_at', 'student.last_name', 'student.first_name']}
-    ];
+    $scope.availableSortingCriterias = [ 'messages', 'name', 'progress', 'last_submission_date' ];
```
Comment (julian-berbel) [inline src/scripts/controllers/guide-progress-controller.js]: just nitpicking, but criteria is already a plural form. The singular form is actually criterion.

**Comment by julian-berbel** [comment]
Looks good!

File: src/scripts/controllers/guide-progress-controller.js (hunk 1 above)
Comment (flbulgarelli) [reply src/scripts/controllers/guide-progress-controller.js]: @fedescarpa

Comment (fedescarpa) [reply src/scripts/controllers/guide-progress-controller.js]: hahahahhaa ok, you know my english is so bad

**Comment by fedescarpa** [comment]
Done!

**Pull request merged**